manuscript/100_Analysis_In_TDD.md

```diff
@@ -15,7 +15,7 @@ Thus, for TDD to be about analysis, it has to fulfill two conditions:
 
 In the story about Johnny, Benjamin and Jane, I�included a�part where they analyze requirements using concrete examples. Johnny explained that this is a�part of a�technique called Acceptance Test-Driven Development. The process followed by the three characters fulfilled both mentioned conditions for a�process to be analytical. But what about TDD itself?
 
-Actually, I�used parts of ATDD process in the story to make the analysis part more obvious, but similar things happen at pure technical levels. For example, when starting development with a�failing application-wide Statement (we will talk about levels of granularity of Statements later. For now the only thing you need to know is that the so called �unit tests level" is not the only level of granularity we write Statements on), we may encounter a�situation where we need to call a�web method and assert its result. This makes us think: what should this method be called? What are the scenarios supported? What do I�need to get out of it? How should its user be notified about errors? Many times, this leads us to either a�conversation (if there is another stakeholder that needs to be involved in the decision) or rethinking our assumptions. This is how we gain a�better understanding of the topic we are analyzing, which makes TDD fulfill the first of the two requirements for it to be an analysis method.
```

Comment:
I think I still need something that marks this paragraph as an answer to the question asked in the previous one. Leaving just "I used parts of ATDD process in the story to make ..." leaves me confused whether this is the answer or a different topic. Can we use something more fitting than "actually" to make this clear?